shared/chat/delete-history-warning/container.tsx

```diff
@@ -19,6 +19,7 @@ export default Container.connect(
         Constants.noConversationIDKey
       )
       dispatch(RouteTreeGen.createNavigateUp())
+      dispatch(RouteTreeGen.createNavigateUp())
```

Comment:
have you tried `dispatch(RouteTreeGen.createClearModals())` instead of two navigate up's?

Reply:
oh good call!